ccd-definition/CaseEvent/CareSupervision/MultiState-nonprod.json

```diff
@@ -0,0 +1,19 @@
+[
```

Comment:
I would add a comment to this and the one being deprecated noting that they should be moved when the toggled on. I use something like
```
"Comment": "To merge/delete when toggling on FPLA-1592"
```